usr/src/test/zfs-tests/tests/functional/zvol/zvol_swap/zvol_swap_004_pos.ksh

```diff
@@ -44,8 +45,18 @@
 
 verify_runnable "global"
 
+function cleanup
+{
+	if is_swap_inuse $swapname ; then
+		log_must $SWAP -d $swapname
```

Comment:
i have added this function based on another function in zfs tests - for consistency and good reading of what we try to do